string name
int64 newStock
---
string feedback
int64 jumlah